src/components/ScrollingIcon/ScrollingIcon.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useEffect, useState } from "react";
+import { BsFillMouseFill, BsMouse } from "react-icons/bs";
+
+export const ScrollingIcon = ({ scrollTimeoutValue }) => {
+  const [isScroll, setIsScroll] = useState(false);
+  const scrollTimeoutDuration = scrollTimeoutValue; // Duration after which isScroll will be set to false
+
+  const handleScroll = (e) => {
+    if (e.deltaY !== 0) {
+      setIsScroll(true);
+      // Clear existing timeout
+      if (window.scrollTimeout) {
+        clearTimeout(window.scrollTimeout);
+      }
+      // Set a new timeout to reset isScroll back to false
+      window.scrollTimeout = setTimeout(() => {
```

Comment:
Is there any reason why you're using window to store the value of scrollTimeout instead of going for something like a useRef?